### PR TITLE
Add assets for 1.20

### LINKED ISSUE #3244: assets 1.20
Olympus: https://www.coingecko.com/en/coins/olympus

### LINKED ISSUE #3245: Add Karura (KAR) to tracked assets
## Abstract

Add Karura asset both on exchanges and on Kusama blockchain.

https://www.coingecko.com/en/coins/karura

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #3244
Closes #3245

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: rotkehlchen/assets/asset.py [hunk 1]
```diff
@@ -236,6 +238,11 @@ def deserialize_from_db(cls, entry: UnderlyingTokenDBTuple) -> 'UnderlyingToken'
     strethaddress_to_identifier('0xbC396689893D065F41bc2C6EcbeE5e0085233447'): 'PERP',
     strethaddress_to_identifier('0xD417144312DbF50465b1C641d016962017Ef6240'): 'CQT',
     strethaddress_to_identifier('0xF5D669627376EBd411E34b98F19C868c8ABA5ADA'): 'AXS',
+    'WBTC': 'WBTC',
```
Comment (LefterisJP) [inline rotkehlchen/assets/asset.py:241]: why is this needed?

File: rotkehlchen/assets/asset.py [hunk 2]
```diff
@@ -236,6 +238,11 @@ def deserialize_from_db(cls, entry: UnderlyingTokenDBTuple) -> 'UnderlyingToken'
...
+    strethaddress_to_identifier('0x491604c0FDF08347Dd1fa4Ee062a822A5DD06B5D'): 'CTSI',
+    strethaddress_to_identifier('0xBBbbCA6A901c926F240b89EacB641d8Aec7AEafD'): 'LRC',
+    'KAR': 'KAR',
```
Comment (LefterisJP) [inline rotkehlchen/assets/asset.py:244]: why is this needed? Do we have multiple ones where the symbol is KAR? I mean it's not wrong to be explicit in case we ever get another token with symbol KAR, but this is not what we have been doing so far.

File: rotkehlchen/chain/ethereum/defi/zerionsdk.py [hunk 3]
```diff
@@ -128,6 +128,9 @@
     'Cozy',
     'Cozy • Yearn',
     'Cozy • Compound',
+    'Unagii',
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/defi/zerionsdk.py:131]: These 3 require frontend changes. Take a look for example here: https://github.com/rotki/rotki/pull/1991/files

**Review by LefterisJP — COMMENTED** [review]
Some comments

File: rotkehlchen/assets/asset.py (hunk 2 above)
Comment (yabirgb) [reply rotkehlchen/assets/asset.py:244]: For kraken we expect all of the assets to be in that list https://github.com/rotki/rotki/blob/develop/rotkehlchen/tests/exchanges/test_kraken.py#L55

Comment (LefterisJP) [reply rotkehlchen/assets/asset.py:244]: oh it was kraken. Sorry I missed it.

Comment (LefterisJP) [reply rotkehlchen/assets/asset.py:244]: It's due to the way github shows the diff

**Review by LefterisJP — APPROVED** [review]
Great. Thanks @yabirgb . LGTM and all green. So pretty. :green_circle:
